### DISCUSSION TITLE
How can I configure a file on installation stage?

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#1821] [Q&A]
怎样在on_install的时候根据target:installdir()来替换configfile的变量？使用自定义的.pc文件和.cmake文件都需要用到这个功能，毕竟自带的rule不能覆盖所有情况

**Comment by waruqi** [comment]
configfile 是 xmake config 阶段就生成完了，你只能在 on_load 阶段设置它们。。

如果要 on_install 还要去替换configfile，你只能自己 io.replace 自己处理下文件
